modules/Bio/EnsEMBL/Variation/Variation.pm

```diff
@@ -433,6 +433,32 @@ sub get_all_synonyms {
 }
 
 
+=head2 get_variation_synonyms
+
+  Arg [1]    : none
+  Example    : my @var_synonyms = $v->get_variation_synonyms;
+  Description: Retrieves a list of all the variation synonyms of this
+               Variation. The output includes the source and the synonyms.
+  Returntype : Listref of strings
+  Exceptions : none
+  Caller     : general
+  Status     : Stable
+
+=cut
+
+sub get_variation_synonyms {
```

Comment:
We already have the subroutines `get_all_synonyms` and `get_all_synonym_sources`, do we need this extra one?